pom.xml

```diff
@@ -46,9 +46,7 @@
         <awaitility.version>4.0.1</awaitility.version>
         <com.h2database.h2.version>2.2.224</com.h2database.h2.version>
         <commons-codec.version>1.15</commons-codec.version>
-        <commons-configuration.version>1.10</commons-configuration.version>
         <commons-io.version>2.11.0</commons-io.version>
-        <commons-lang3.version>3.12.0</commons-lang3.version>
```

Comment:
Why did you remove the variable instead of updating the version here? I think it is clearer to have all version numbers in one place instead of gathering from all over the code. Wouldn't you agree?

Reply:
This (and a lot of other) variable is not needed anymore as the usage is on one place. In the past the used versions was moved into the main pom.xml file and the module pom.xml files rely on the version information of the main pom.file. I can re-add this variable if you prefer this.

Reply:
I re-added the variable. Cleanup such one-use variable should be done in future.